scripts/paymaster-load-test/src/generator.rs

```diff
@@ -75,6 +77,22 @@ impl TransactionGenerator {
         Ok(tx)
     }
 
+    /// Generate a valid memo transaction
+    fn generate_valid_memo(&self, blockhash: Hash) -> Result<VersionedTransaction> {
+        let instructions = vec![
+            ComputeBudgetInstruction::set_compute_unit_limit(10_000),
+            spl_memo::build_memo(random::<u64>().to_string().as_bytes(), &[]),
+        ];
+
+        let message =
+            v0::Message::try_compile(&self.sponsor_pubkey, &instructions, &[], blockhash)?;
+
+        Ok(VersionedTransaction {
+            signatures: vec![Default::default(); 1],
```

Comment:
nit for devx: maybe a comment that this is the sponsor signature so doesn't need to be signed when submitted to the paymaster? we do have that comment above so maybe there is a way to modularize and have it once?